src/Form.php

```diff
@@ -80,6 +84,17 @@ public function clearErrors(?string $attribute = null): void
         }
     }
 
+    public function getAttributeValue(string $attribute)
```

Comment:
Since we're using magic getters, was there ever a consideration **not** to use class methods starting with `get...`? 
I.e. also `getError()` but rather something like `resolveError()` or `retrieveError()`, might be late to the party here, but I stumbled upon that in Yii2 quite sometimes.

CC: @samdark

Reply:
ok no problem i will add a suffix parameter to the methods so there will be no problem using any name

Reply:
Done.

Reply:
I think it's rarely case. Getters and setters are exist in all the world. If you want to use non-standart getter method name, you can just call the getter in your custom method.